Adding `Contribute` section (#114)
ansys/pymapdl-examples: doc/source/contributing/index.rst

    git clone https://github.com/ansys/pymapdl-examples
    cd pymapdl-examples

Post issues
-----------

Use the `PyMAPDL Examples Issues <pymapdl_examples_issues_>`_ page to submit questions,
report bugs, and request new features. When possible, use these issue
templates:

* Bug report template
* Feature request template

If your issue does not fit into one of these categories, create your own issue.

To reach the project support team, email `pyansys.core@ansys.com <pyansys.core@ansys.com>`_.

Python virtual environment
--------------------------

The use of Python virtual environment is recommended. To create one, run the following commands::

    python -m venv .venv
     .\.venv\Scripts\activate

To deactivate it, run::
    
    deactivate

Build documentation
-------------------

To build the PyMAPDL Examples documentation locally, in the root directory of the repository, run::
    
    pip install -r .\requirements\requirements_doc.txt
    .\doc\make.bat html

Documentation for the latest stable release of PyMAPDL Examples is hosted at
`PyMAPDL Examples Documentation <pymapdl_examples_docs_>`_.

> In the upper right corner of the documentation’s title bar, there is an option for
> switching from viewing the documentation for the latest stable release to viewing
> the documentation for the development version or previously released versions.
> 

Reviewer: ```suggestion
```

There is no switcher for versions in the PyMAPDL Examples docs. And it is not needed.

Adhere to code style
--------------------

PyMAPDL Examples follows the PEP8 standard as outlined in the `PyAnsys Development Guide
<dev_guide_pyansys_>`_ and implements style checking using
`pre-commit <precommit_>`_.

To ensure your code meets minimum code styling standards, first, you need to install
the ``pre-commit`` library::

  pip install pre-commit

Once done, code styling standards can be checked with the following code::

  pre-commit run --all-files

You can also install this as a pre-commit hook by running this code::

  pre-commit install

> This way, it's not possible for you to push code that fails the style checks::

Reviewer: ```suggestion
This way, it is not possible for you to push code that fails the style checks::
```

Contribute
==========

Overall guidance on contributing to a PyAnsys library appears in the
`Contributing <dev_guide_contributing_>`_ topic
in the *PyAnsys Developer's Guide*. Ensure that you are thoroughly familiar

> with this guide before attempting to contribute to PyMAPDL Examples.

Reviewer 2: ```suggestion
with this guide before attempting to contribute to PyMAPDL.
```

The following contribution information is specific to PyMAPDL Examples.

Clone the repository
--------------------

Run this code to clone and install the latest version of PyMAPDL Examples in development mode::

    git clone https://github.com/ansys/pymapdl-examples
    cd pymapdl-examples

Post issues
-----------

Use the `PyMAPDL Examples Issues <pymapdl_examples_issues_>`_ page to submit questions,
report bugs, and request new features. When possible, use these issue
templates:

* Bug report template
* Feature request template

If your issue does not fit into one of these categories, create your own issue.

To reach the project support team, email `pyansys.core@ansys.com <pyansys.core@ansys.com>`_.

Python virtual environment
--------------------------

> The use of Python virtual environment is recommended. To create one, run the following commands::

Reviewer 2: ```suggestion
The use of a Python virtual environment is recommended. To create one, run the following commands::
```

    python -m venv .venv
     .\.venv\Scripts\activate

> To deactivate it, run::

Reviewer 2: ```suggestion
To deactivate the virtual environment, run::
```

    
    deactivate

Build documentation
-------------------

> To build the PyMAPDL Examples documentation locally, in the root directory of the repository, run::

Reviewer 2: ```suggestion
To build the PyMAPDL Examples documentation locally, in the root directory of the repository, run
these commands::
```

    
    pip install -r .\requirements\requirements_doc.txt
    .\doc\make.bat html

Documentation for the latest stable release of PyMAPDL Examples is hosted at
`PyMAPDL Examples Documentation <pymapdl_examples_docs_>`_.

> 

Reviewer 2: Do we need this blurb about how to view the documentation for other releases:

In the upper right corner of the documentation’s title bar, there is an option for switching from viewing the documentation for the latest stable release to viewing the documentation for the development version or previously released versions.

Author: It's not possible to switch from a version to another for this repository as it only contains examples . However, you're right that this section is unnecessary: the published version corresponds to the main branch.

Adhere to code style
--------------------

PyMAPDL Examples follows the PEP8 standard as outlined in the `PyAnsys Development Guide
<dev_guide_pyansys_>`_ and implements style checking using
`pre-commit <precommit_>`_.

To ensure your code meets minimum code styling standards, first, you need to install

> the ``pre-commit`` library::

Reviewer 2: ```suggestion
the ``pre-commit`` library with this command::
```
Library or package?

  pip install pre-commit

> Once done, code styling standards can be checked with the following code::

Reviewer 2: ```suggestion
Once installed, you can verify that the code meets code style standards by running this comand::
```

  pre-commit run --all-files

> You can also install this as a pre-commit hook by running this code::

Reviewer 2: ```suggestion
You can also install this as a pre-commit hook by running this command::
```

Contribute
==========

Overall guidance on contributing to a PyAnsys library appears in the
`Contributing <dev_guide_contributing_>`_ topic
in the *PyAnsys Developer's Guide*. Ensure that you are thoroughly familiar
with this guide before attempting to contribute to PyMAPDL Examples.

The following contribution information is specific to PyMAPDL Examples.

Clone the repository
--------------------

Run this code to clone and install the latest version of PyMAPDL Examples in development mode::

    git clone https://github.com/ansys/pymapdl-examples
    cd pymapdl-examples

Post issues
-----------

Use the `PyMAPDL Examples Issues <pymapdl_examples_issues_>`_ page to submit questions,
report bugs, and request new features. When possible, use these issue
templates:

* Bug report template
* Feature request template

> If your issue does not fit into one of these categories, create your own issue.

Reviewer 2: ```suggestion
If your issue does not fit into one of these template categories, create your own issue.
```

To reach the project support team, email `pyansys.core@ansys.com <pyansys.core@ansys.com>`_.

Python virtual environment
--------------------------

> 
> The use of a Python virtual environment is recommended. To create one, run the following commands::

Reviewer 2: ```suggestion

The use of a Python `virtual environment <https://docs.python.org/3/library/venv.html>`_ is recommended.
To create one, run the following commands::
```

    python -m venv .venv
     .\.venv\Scripts\activate

To deactivate the virtual environment, run this command::
    
    deactivate

Build documentation
-------------------

To build the PyMAPDL Examples documentation locally, in the root directory of the repository,
run these commands::
    
    pip install -r .\requirements\requirements_doc.txt
    .\doc\make.bat html

Documentation for the latest stable release of PyMAPDL Examples is hosted at
`PyMAPDL Examples Documentation <pymapdl_examples_docs_>`_.

Adhere to code style
--------------------

PyMAPDL Examples follows the PEP8 standard as outlined in the `PyAnsys Development Guide
<dev_guide_pyansys_>`_ and implements style checking using
`pre-commit <precommit_>`_.

> To ensure your code meets minimum code styling standards, first, you need to install
> the ``pre-commit`` library with this command::

Reviewer 2: ```suggestion
``pre-commit`` is a multi-language package manager for pre-commit hooks. To ensure
that your code meets minimum code styling standards, install ``pre-commit`` with
this command::
```
